Makefile.am

```diff
@@ -134,8 +134,8 @@ TESTS = \
 		tests/bser.t \
 		tests/ignore.t \
 		tests/pending.t \
-		tests/string.t \
 		tests/log.t \
+		tests/string.t \
```

Comment:
please revert the changes in this file; since you're now adding to the existing test, this pull request doesn't need to make changes here :-)